lib/crowbar/init/application.rb

```diff
@@ -378,6 +388,9 @@ class Application < Sinatra::Base
       # api_version "2.0"
       post "/api/upgrade/connect" do
         api_constraint(2.0)
+        upgrade_status = ::Crowbar::UpgradeStatus.new(logger)
+        upgrade_status.start_step
```

Comment:
Please change to `upgrade_status.start_step :database` once https://github.com/crowbar/crowbar-core/pull/816 is merged